lib/search/query_components/query.rb

```diff
@@ -38,17 +55,31 @@ def query_hash
       }
     end
 
-    def base_query
-      return { match_all: {} } if search_term.nil?
-
-      if search_params.enable_new_weighting?
-        core_query = QueryComponents::TextQuery.new(search_params).payload
-      else
-        core_query = QueryComponents::CoreQuery.new(search_params).payload
-      end
-
-      boosted_query = QueryComponents::Booster.new(search_params).wrap(core_query)
-      QueryComponents::Popularity.new(search_params).wrap(boosted_query)
+    def more_like_this_query_hash
+      {
+        more_like_this: {
+          docs: [
+            {
+              _index: :mainstream,
+              _type: "edition",
```

Comment:
I've removed the index since that will make it search across all indices. However, if I remove the type then the document ID becomes ambiguous. We either have to include blocks for each document type, or somehow make document IDs unique across types.